package.json

```diff
@@ -91,17 +91,17 @@
   },
   "devDependencies": {
     "chai": "~3.5.0",
-    "chai-as-promised": "^5.3.0",
+    "chai-as-promised": "^6.0.0",
```

Comment:
Is chai-as-promised actually used? There is one file that requires it, but I don't think the features of chai-as-promised are even used in it.